source/howto/setup/shared-cache.rst

```diff
@@ -0,0 +1,74 @@
+.. meta::
+    :description: Setting up a shared network cache for your Valohai workers 
+
+.. _setup-shared-cache:
+
+Setting up a shared cache for Valohai workers
+##############################################
+
+Most of your Valohai machine learning jobs have a set of input files that are downloaded from a cloud object storage like AWS S3, Azure Blob Storage, or GCP Cloud Storage. This data is downloaded on each of the virtual machines that are running your job.
+
+By default each Valohai worker (virtual machine) will have it's own cache where the downloaded data is stored. When the machine is no longer used it get scaled down, and with it the cache gets removed, and the next time a machine gets scaled up it will download the input files to it's own cache.
```

Comment:
```suggestion
By default each Valohai worker (virtual machine) will have its own cache where the downloaded data is stored.
When the machine is no longer used (after a configurable grace period) it gets scaled down, and with it the local cache gets removed.
The next time a machine gets scaled up it will download the input files to its own cache.
```